src/stories/Meter.stories.js

```diff
@@ -0,0 +1,120 @@
+import { html } from 'lit';
+import '../components/chart';
+import argTypes from '../common/config/chartArgTypes';
+
+export default {
+  title: 'Charts/Meter',
+  component: 'kd-chart',
+  decorators: [
+    (story) => html` <div style="max-width: 600px;">${story()}</div> `,
+  ],
+  parameters: {
+    design: {
+      type: 'figma',
+      url: 'https://www.figma.com/file/uwR7B1xbaRXA5spwPvzzFO/Florence-Release?node-id=2051%3A93&mode=dev',
+    },
+  },
+  argTypes: argTypes,
+};
+
+const args = {
+  chartTitle: 'Meter Chart',
+  description: 'Chart description.',
+  labels: ['LOW', 'MED', 'HIGH'],
+  datasets: [
+    {
+      data: [30, 30, 30],
+      borderWidth: 2,
```

Comment:
Move to `/config/chartTypes/meter.js`

Reply:
`data: [30,30,30] `will be part of stories correct ?

Reply:
Yeah data is always user supplied

Reply:
ok then it's here only inside stories.js